frontend/src/components/pages/CampCreation/StepperNavigation/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Box, Center } from "@chakra-ui/react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCampground,
+  faCalendarDays,
+  faCircleInfo,
+} from "@fortawesome/free-solid-svg-icons";
+import StepperTab from "./StepperTab";
+
+export enum CampCreationPages {
+  CampCreationDetailsPage,
+  ScheduleSessionsPage,
+  RegistrationFormPage,
+}
+
+type StepperProps = {
+  currentPage: CampCreationPages;
+  setCurrentPage: React.Dispatch<React.SetStateAction<CampCreationPages>>;
+  isCampDetailsFilled: boolean;
+  isScheduleSessionsFilled: boolean;
+  isRegistrationFormFilled: boolean;
+};
+
+const Stepper = ({
+  currentPage,
+  setCurrentPage,
+  isCampDetailsFilled,
+  isScheduleSessionsFilled,
+  isRegistrationFormFilled,
```

Comment:
We aren't currently using this prop in the stepper but we probably will need it later (when we try to submit the camp to the db)

Reply:
would we need this prop in the Stepper component though? I think it would be useful in the footer so we can conditionally render the `publish camp` button or use it for validation